TYPO3.Flow/Classes/TYPO3/Flow/Cache/Backend/SimpleFileBackend.php

```diff
@@ -260,14 +255,23 @@ public function remove($entryIdentifier)
 
         $pathAndFilename = $this->cacheDirectory . $entryIdentifier . $this->cacheEntryFileExtension;
 
-        try {
-            $lock = new Lock($pathAndFilename);
-            unlink($pathAndFilename);
-            $lock->release();
-        } catch (\Exception $exception) {
+        if (strncasecmp(PHP_OS, 'WIN', 3) == 0) {
+            // On Windows, unlinking a locked/opened file will not work, so we just attempt the delete straight away.
+            // In the worst case, the unlink will just fail due to concurrent access and the caller needs to deal with that.
+            return @unlink($pathAndFilename);
+        }
+
+        $file = @fopen($pathAndFilename, 'r');
+        if ($file === false) {
```

Comment:
I have concerns about the shut-up-operator – wouldn't `try`/`catch` work here?

It's a bit of history repeating – we had this several times in and out in similar places, see also https://karsten.dambekalns.de/blog/a-story-about-include-once.html

So, don't rush this until we're really sure.